src/banner/banner.stories.ts

```diff
@@ -32,11 +31,35 @@ class BannnerStory {
 	}
 }
 
+@Component({
+	selector: "app-toast-story",
+	template: `
+		<button class="bx--btn bx--btn--primary" (click)="showToast()">Show info toast</button>
+		<div class="banner-container"></div>
+	`,
+	providers: [BannerService]
+})
+class ToastStory {
+
```

Comment:
Empty line.

Reply:
I'm changing it, but I'm not sure I like the constructors hugging the class declaration like that